transformation/src/main/scala/org/broadinstitute/monster/hca/PostProcess.scala

```diff
@@ -1,13 +1,15 @@
 package org.broadinstitute.monster.hca
 
-import com.spotify.scio.{ScioMetrics, ScioResult}
-import org.apache.beam.sdk.metrics.Counter
+import com.spotify.scio.ScioResult
 
 object PostProcess {
-  val errorCount: Counter = ScioMetrics.counter("errorCount")
 
   def postProcess(result: ScioResult): Unit = {
-    result.counter(errorCount).committed.fold(())(count => if (count > 0) throw new HcaFailException)
+    result.allCounters.foreach {
+      case (name, count) =>
+        if (name.getName == "errorCount")
```

Comment:
Maybe put `"errorCount"` into a declared constant to make it clear that it's the same thing everywhere this string is repeated.

Reply:
Good call. I was annoyed to have to do it this way, apparently when we use the dataflow runner the metric has to be declared inside a block of code that is run in the pipeilne (that restriction doesn't apply to the direct runner), but a constant for the string is good.